test/api/multiple-results-event.test.js

```diff
@@ -0,0 +1,133 @@
+const { isString } = require('lodash');
+const Chai = require('chai');
+const sinon = require('sinon');
+const ChaiAsPromised = require('chai-as-promised');
+
+const Event = require('../../src/api/multiple-results-event');
+const Mocks = require('./mock/multiple-results-event');
+
+Chai.use(ChaiAsPromised);
+const assert = Chai.assert;
+const expect = Chai.expect;
+
+const eventAddress = '0x8b2b1838efff78e5ed9e00d95d9ef071f2c27be6';
+const address = '0x939592864C0Bd3355B2D54e4fA2203E8343B6d6a';
+
+describe('api/multiple-results-event', () => {
+  const contractAddress = 'a5b27c03e76d4cf10928120439fa96181f07520c';
+  let stubGetContract;
+
+  beforeEach(() => {
+      stubGetContract = sinon.stub(Event, 'getContract').callsFake((eventAddress) => {
+        if (eventAddress !== '0x8b2b1838efff78e5ed9e00d95d9ef071f2c27be6') return undefined;
+        return {
+          methods: {
+            calculateWinnings: (address) => {
```

Comment:
lets stub each contract method as well.
https://stackoverflow.com/a/44154342/2917742

Code not tested, just as example.
```
const stubCalculateWinnings = sinon.stub().returns('10200000000');  
sinon.stub().returns({
  methods: {
    calculateWinnings: stubCalculateWinnings,
    ...
  }
}
```

We aren't concerned about passing in the correct address or validating the contract instance, but more about getting the mock data from the API.

Reply:
Think we should test if address is not listed, then no winnings?

Reply:
No need to test if the address is not listed. That is a test for the contract level. This test is more for the formatting of the data from the contract.